package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "hanawilo-project",
+  "lockfileVersion": 2,
+  "requires": true,
```

Comment:
There should not be any changes to the `package-lock.json` file since no new package has been added.